qiskit_experiments/test/t1_backend.py

```diff
@@ -108,7 +108,7 @@ def run(self, run_input, **options):
                         prob1[qubit] = 1 - prob1[qubit]
                     elif op.name == "delay":
                         delay = op.params[0]
-                        prob1[qubit] = prob1[qubit] * np.exp(-delay / self._t1[qubit])
+                        prob1[qubit] = prob1[qubit] * np.exp(-delay / self._t1)
```

Comment:
The `[qubit]` term should remain there. Different qubits can have different T1s and the user has to provide an array of T1s when initializing the backend.